runner.go

```diff
@@ -124,19 +199,31 @@ no_proxy = %s
 
 var cmd *exec.Cmd
 
-var ctx, stop = signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+var ctx, _ = signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
 
 func RunSplunk() bool {
+	name := expandEnv(SplunkPath)
 	args := []string{"start", "--answer-yes", "--nodaemon"}
 	args = append(args, os.Args[1:]...)
-	cmd = exec.CommandContext(ctx, os.ExpandEnv(SplunkPath), args...)
+	cmd = exec.CommandContext(ctx, name, args...)
 	cmd.Stdout = os.Stderr
 	cmd.Stderr = os.Stderr
-	cmd.Start()
-	cmd.Wait()
+	if cmd.Start() != nil {
+		return false
+	}
+	if cmd.Wait() != nil {
+		return false
+	}
```

Comment:
Thank you for these changes. I still think we should be able to do something with returning the errors instead of a bool, but this will appease The Linter.